src/generators/java/mekanism/generators/common/config/GeneratorsConfig.java

```diff
@@ -101,6 +106,16 @@
               .defineList("windGenerationDimBlacklist", new ArrayList<>(), o -> o instanceof String && ResourceLocation.tryCreate(((String) o).toLowerCase(Locale.ROOT)) != null));
         builder.pop();
 
+        builder.comment("Fusion Settings").push(FUSION_CATEGORY);
+        fusionThermocoupleEfficiency = CachedDoubleValue.wrap(this, builder.comment("The fraction of the heat dissipated from the case that is converted to Joules.")
+               .defineInRange("fusionThermocoupleEfficiency", 0.05D, 0D, 1D));
```

Comment:
The fusion prefix from these config options should be removed for example `thermocoupleEfficiency`, `casingThermalConductivity`, `waterHeatingRatio` as they are already in the fusion category so the config path is evident they are for fusion. The variable names however should be left as is.

Reply:
Done